src/PJ_pipeline.c

```diff
@@ -0,0 +1,547 @@
+/***********************************************************************
+
+               Transformation pipeline manager
+
+                 Thomas Knudsen, 2016-05-20/2016-11-14
+
+************************************************************************
+
+**The Problem**
```

Comment:
Works for me too. Most of the text can be used as a general introduction in the docs as well.